frontend/src/metabase/setup/components/EmbeddingSetup/steps/FinalStep.tsx

```diff
@@ -25,22 +24,15 @@ import type { StepProps } from "./embeddingSetupSteps";
 
 export const FinalStep = ({ nextStep }: StepProps) => {
   const { url: docsUrl } = useDocsUrl("embedding/interactive-embedding");
-  const { createdDashboardIds, trackEmbeddingSetupClick } = useEmbeddingSetup();
-
-  const { loading, value: dashboards } = useAsync(async () => {
-    const dashboardPromises = createdDashboardIds.map((id) =>
-      fetch(`/api/dashboard/${id}`).then((res) => res.json()),
-    );
-    return Promise.all(dashboardPromises);
-  }, [createdDashboardIds]);
+  const { createdDashboard, trackEmbeddingSetupClick } = useEmbeddingSetup();
 
   const getEmbedCode = (url: string) => {
     return `<iframe src="${url}" width="800px" height="500px" />`;
   };
 
   const tabs = useMemo(
     () => [
-      ...(dashboards ?? []).map((dashboard: Dashboard) => ({
+      ...createdDashboard.map((dashboard: Dashboard) => ({
```

Comment:
Should this be `createdDashboards` since each item in the list has its own dashboard inside?